rust/agama-lib/src/manager.rs

```diff
@@ -85,6 +88,72 @@ impl TryFrom<u32> for InstallationPhase {
     }
 }
 
+/// Finish method
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone, Copy, utoipa::ToSchema)]
+pub enum FinishMethod {
+    #[serde(rename = "halt")]
+    // Halt the system
+    Halt,
+    #[serde(rename = "reboot")]
+    // Reboots the system
+    Reboot,
+    #[serde(rename = "stop")]
+    Stop,
```

Comment:
What is the difference between `stop` and `halt`?

Reply:
Will improve the documentation, stop was requested to do nothing, that is, stop at the end of the installation while halt .... halt the machine at the end of the installation.